services/director/src/simcore_service_director/producer.py

```diff
@@ -997,6 +1008,26 @@ async def get_service_details(app: web.Application, node_uuid: str) -> Dict:
             ) from err
 
 
+@retry(
```

Comment:
So this is just a retry if the connectivity is missing? I don't recall exactly why the connectivity went missing. Could you please explain that?

Reply:
Good question. In short it is not strictly necessary but we experienced a strange situation in which the service somehow lost connection.  In addition, the call to this entrypoint in the front-end is done with a fire&forget so it is a small extra measure to consolidate this call. IMO it does not harm.